src/Views/DriversView.vala

```diff
@@ -0,0 +1,114 @@
+/*-
+ * Copyright 2023 elementary, Inc. (https://elementary.io)
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+ public class DriversView : AbstractInstallerView {
+    public signal void next_step ();
+
+    construct {
+        var image = new Gtk.Image.from_icon_name ("application-x-executable", Gtk.IconSize.DIALOG) {
+            pixel_size = 128,
+            valign = Gtk.Align.END
+        };
+
+        var title_label = new Gtk.Label (_("Install Drivers")) {
+            valign = Gtk.Align.START
+        };
+        title_label.get_style_context ().add_class (Granite.STYLE_CLASS_H2_LABEL);
+
+        var description_label = new Gtk.Label (_("Proprietary drivers are required for some devices, such as Wi-Fi adapters or graphics cards, to work properly.")) {
+            max_width_chars = 1, // Make Gtk wrap, but not expand the window
+            wrap = true,
+            xalign = 0
+        };
+        description_label.get_style_context ().add_class (Granite.STYLE_CLASS_H3_LABEL);
+
+        var warning_image = new Gtk.Image.from_icon_name ("security-low-symbolic", LARGE_TOOLBAR);
+
+        var warning_label = new Gtk.Label (_("Proprietary drivers contain closed-source components. You may prefer not to install them due to hardware compatibility issues.")) {
+            hexpand = true,
+            max_width_chars = 1, // Make Gtk wrap, but not expand the window
+            wrap = true,
+            xalign = 0
+        };
+
+        var legal_image = new Gtk.Image.from_icon_name ("text-x-copying-symbolic", LARGE_TOOLBAR);
+
+        var legal_label = new Gtk.Label (_("These drivers come with their own licensing terms. For that reason, we cannot redistribute them freely.")) {
```

Comment:
I think redistribution is an issue with codecs, but I'm not sure that it is with drivers. I'm having a hard time finding out what these licenses actually are though and how to review them

```suggestion
        var legal_label = new Gtk.Label (_("Proprietary drivers are subject to their own license terms. By installing, you agree to their terms of use.")) {
```